pyproject.toml

```diff
@@ -13,7 +13,7 @@ authors = [
 ]
 description = "Experiment management and market simulators by Element Finance"
 readme = "README.md"
-requires-python = ">=3.8"
+requires-python = ">=3.9, <=3.10"
```

Comment:
If we're going to continue to support colab, then we should pin the project to 3.9 and not 3.10. I'd prefer to sunset colab and bump up to 3.10, but that's outside of the scope of this PR.

Reply:
I've been using 3.10 locally

Reply:
switched to 3.9 for Colab compatibility